### PR TITLE
Release v1.2.0

### PR DESCRIPTION
**Pull request by rishijain**
Release v1.2.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# main [(unreleased)](https://github.com/fastruby/next_rails/compare/v1.1.0...main)
+# main [(unreleased)](https://github.com/fastruby/next_rails/compare/v1.2.0...main)
+
+# v1.2.0 / 2022-08-12 [(commits)](https://github.com/fastruby/next_rails/compare/v1.1.0...v1.2.0)
```
Comment (fbuys) [inline CHANGELOG.md:3]: Looks great, one small thing related to your PR, can we align line 5?
It looks like it has an extra space in front.

**Review by fbuys — COMMENTED** [review]
One small comment, after that I think it is good to go!

File: CHANGELOG.md (hunk 1 above)
Comment (rishijain) [reply CHANGELOG.md:3]: hey @fbuys I looked it up on my editor and here as well https://github.com/fastruby/next_rails/blob/f534dff96eea533e41ec7c00dfaa0f2009dad12b/CHANGELOG.md

It looks aligned to me. 

I don't know why it looks off on the diff though.

**Review by kindoflew — APPROVED** [review]
LGTM!

**Pull request merged**